.buildkite/scripts/publish.sh

```diff
@@ -15,3 +17,10 @@ aws cloudfront create-invalidation \
     --distribution-id "${KAIZEN_DISTRIBUTION_ID}" \
     --paths "${KAIZEN_BASE_PATH:-/}*" \
     --output text
+
+echo "Reporting the deployed URL via the GitHub status API"
+
+commit_hash=$(git rev-parse --verify HEAD)
+curl --fail -s "https://api.github.com/repos/cultureamp/kaizen-design-system/statuses/$commit_hash" \
+  --header "authorization: Bearer $GITHUB_STATUS_TOKEN" \
+  --data '{"state": "success", "target_url": "https://'"$destination"'", "description": "Branch preview", "context": "Branch preview"}'
```

Comment:
This reports "***Branch preview** — Branch preview*" at the moment. Happy to approve, but this could also include the URL to make it clear what it's talking about (although confusingly the URL won't be a link...)

```suggestion
  --data '{"state": "success", "target_url": "https://'"$destination"'", "description": "'"$destination"'", "context": "Branch preview"}'
```